bentoml/cli/__init__.py

```diff
@@ -91,6 +115,20 @@ def serve(port, archive_path=installed_archive_path):
         server = BentoAPIServer(model_service, port=port)
         server.start()
 
+    # Example Usage: bentoml serve-gunicorn ./SAVED_ARCHIVE_PATH --port=PORT --workers=WORKERS
+    @bentoml_cli.command()
+    @conditional_argument(installed_archive_path is None, 'archive-path', type=click.STRING)
+    @click.option('-p', '--port', type=click.INT, default=BentoAPIServer._DEFAULT_PORT)
+    @click.option('-w', '--workers', type=click.INT, default=(multiprocessing.cpu_count() // 2) + 1)
```

Comment:
move `multiprocessing.cpu_count() // 2) + 1` out as a separate method and add some comments about the choice here?